src/FlowInstancesPage.tsx

```diff
@@ -99,7 +99,9 @@ export default (): React.ReactElement => {
     return instances;
   }, [astarte.client]);
 
-  const instancesFetcher = useFetch(fetchInstances);
+  const { value: flowInstances, status: refreshStatus, refresh: refreshInstances } = useFetch(
+    fetchInstances,
+  );
```

Comment:
Why we need to spread and rename variables here? We didn't have to in other pages